### DISCUSSION TITLE
[DropdownMenu] Sliding menu

### DISCUSSION THREAD

**Discussion by YutaMoriJP** [radix-ui/primitives#1841] [Help] [answered]
## Question:

Is there an example with Radix UI's Dropdownmenu primitives that would achieve a similar experience to Youtube's Setting behavior? When you click on the video's gear icon a menu pops up, and you can use your keyboard to navigate to the quality and then the right arrow to open the quality selection menu. This experience feels similar to the DropdownMenu. Are there such examples, and is it even possible?

This is pretty close in spirit - https://ariakit.org/examples/menu-slide

**Comment by benoitgrelard** [comment, marked as answer]
Hey @YutaMoriJP,

We don't currently have anything that would support this use-case.

**Reply by YutaMoriJP** [reply]
Thanks for getting back to me! Noted 🙏
